app/models/form/sales/pages/not_retired_value_check.rb

```diff
@@ -0,0 +1,25 @@
+class Form::Sales::Pages::NotRetiredValueCheck < Form::Sales::Pages::Person
```

Comment:
This works exactly like it does for lettings (with slight naming differences because there already were those)